### ISSUE TITLE
Upgrading from 0.31.1 to 0.31.3 breaks builds

### ISSUE THREAD

**Issue by mduleone** [DataDog/dd-trace-js#1278]
**Describe the bug**
Dependabot opened a PR on our repo to upgrade `dd-trace` from 0.31.1 (to 0.31.2, and then) to 0.31.3.

Our build (of a next-js application) is failing in that environment with the following error

```sh
ModuleNotFoundError: Module not found: Error: Can't resolve './packages/dd-trace/browser' in '/tmp/build_825d023a/node_modules/dd-trace'
> Build error occurred
Error: > Build failed because of webpack errors
    at build (/tmp/build_825d023a/node_modules/next/dist/build/index.js:15:918)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
```

**Environment**
This is running in a Heroku review environment running heroku-18/nodejs stack

* **Operation system:** Linux (unsure of flavor, can get that info if it's relevant
* **Node version:** 12.x
* **Tracer version:** 0.31.3

**Comment by rochdev** [comment]
It looks like for some reason `browser.js` was included in previous versions even though it's not explicitly excluded from `.npmignore` and the latest version of `npm` started to ignore it when we release. I've explicitly added the file now and will make another release.

It's worth noting however that 0.31 is the last version with browser support. See the [release notes](https://github.com/DataDog/dd-trace-js/releases/tag/v0.31.0) for more details.

**Comment by rochdev** [comment]
Fix released in 0.31.4.

**Comment by mduleone** [comment]
Appreciate the call out of deprecating browser support @rochdev. I see there's now an `allowedTracingOrigins` option on the RUM init function, where previously we needed to set up tracing ourselves. Will keep this in mind when we go to bump to `0.32.x`
